data/framework-laptop12.tablet

```diff
@@ -0,0 +1,17 @@
+[Device]
+Name=Framework Laptop 12
+Class=ISDV4
+# VID belongs to Ilitek, the touch IC vendor
+# PID is reserved for this Framework Laptop 12
+DeviceMatch=i2c|222a|5539
+# Dimensions of the touch area in inches
+# Actually 10.5x6.6in, rounding up
+Width=11
+Height=7
+IntegratedIn=Display;System;
+Styli=@generic-no-eraser;
+
+[Features]
+Stylus=true
+Touch=true
+Buttons=0
```

Comment:
```suggestion
```

See 2b336bb4f41cae8c836033c6aa5e1c5e995710cb though it's not a hard failure to have it, it's just ignored